integration-tests/test_stdin_to_stdout.py

```diff
@@ -21,3 +23,25 @@ def test_stdin_to_stdout(tmpdir):
     assert result.get("gender") == "M"
 
     os.remove(output_file)
+
+@pytest.mark.xfail(run=False)
```

Comment:
why is it marked with xfail?

Reply:
removed

Reply:
it is related to the filter bug. will be resolved with that PR.